pkg/ebpf/tracer.go

```diff
@@ -625,6 +646,17 @@ func (t *Tracer) isLocalAddress(address util.Address) bool {
 	return ok
 }
 
+func (t *Tracer) resolveConnections(connections ...ConnectionStats) []ConnectionStats {
+	names := t.reverseDNS.Resolve(connections)
+
+	for i := range connections {
+		connections[i].SourceDNS = names[i].Source
```

Comment:
Does the source ever actually get a DNS entry mapped to it?

Reply:
humm, I don't know TBH. but I'd leave this here until we think this through.